extensions/wikia/CategorySelect/CategorySelectController.class.php

```diff
@@ -8,7 +8,7 @@
  * @author Kyle Florence <kflorence@wikia-inc.com>
  */
 
-class CategorySelectController extends WikiaController {
+class CategorySelectController extends WikiaService {
 	const CACHE_TTL_AJAX = 360; // 1 hour
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)